waiter/src/waiter/settings.clj

```diff
@@ -269,7 +271,9 @@
                    :interval-ms 1500}
    :scaling {:autoscaler-interval-ms 1000
              ; throttles the rate at which kill requests are sent to the scheduler
-             :inter-kill-request-wait-time-ms 1000}
+             :inter-kill-request-wait-time-ms 1000
+             :quanta-constraints {:cpus 64
+                                  :mem (* 512 1024)}}
```

Comment:
We need to decide on sane defaults.

Reply:
Is this something left to do in this PR?

Reply:
@sradack thinks the current defaults are reasonable:

> Defaults seem reasonable enough.